### DISCUSSION TITLE
ASCII 3D 렌더러 만들기 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#29] [Announcements]
# ASCII 3D 렌더러 만들기 | kciter.so

필자는 쓸데없지만 재밌는 프로그램을 만들면서 공부하는 것을 좋아한다. 최근에 정신적으로 에너지가 바닥나고 있다는 것을 느껴 작년에 재미로 만들었던  Jazzlang 과 같이 정신적 에너지를 채워줄만한 재미를 느끼고 싶었다. 그래서 무언가 할만한 것을…

https://kciter.so/posts/ascii-3d-renderer

**Comment by ibcylon** [comment]
흥미로운 글이네요

**Comment by BeingApple** [comment]
킹갓개발자;;

**Comment by clucle** [comment]
재밌게 읽었습니다

**Comment by ondaa** [comment]
세상은 넓고 천재는 많다.

**Comment by jyjung** [comment]
멋지네요

**Comment by DevSDK** [comment]
재밌게 읽었습니다 :)

**Comment by ian616** [comment]
같은 프로젝트를 진행해보고 있는데, 최적화는 어떤 부분에서 할 수 있까요? 생각보다 프레임이 너무 안나오네요ㅠㅠ

**Reply by kciter** [reply]
안녕하세요. 질문주신걸 너무 늦게봤네요 ^^;;
자세히 살펴보지 않아 확실한건 아니지만 작업 중인 프로젝트 [저장소](https://github.com/ian616/Ascii3D)를 확인했을 때 리액트를 사용하시는 것 같습니다. 리렌더링이 너무 자주 발생하는 것이 아닌지 확인해보세요.
그리고 제가 관련 코드를 못찾은 것일 수도 있는데 초당 프레임을 고정해서 렌더링 횟수를 줄이는 것도 도움이 될 것 같습니다.
